modules/tinymce/src/core/main/ts/fmt/Preview.ts

```diff
@@ -276,7 +277,7 @@ const getCssText = function (editor: Editor, format) {
   parentFontSize = dom.getStyle(editor.getBody(), 'fontSize', true);
   parentFontSize = /px$/.test(parentFontSize) ? parseInt(parentFontSize, 10) : 0;
 
-  each(previewStyles.split(' '), function (name) {
+  each((previewStyles as string).split(' '), (name: string) => { // Enforcing type as previewStyles may at times contain boolean. This confused typescript.
```

Comment:
I assume you then also want me to change all the instances of `previewStyles === false` to `previewStyles === ''` ?

Reply:
No, you'd just remove that code, as the only place currently does this:
```
if (previewStyles === false) {
  previewStyles = ''
}
```

Reply:
There is also
`previewStyles = format.preview;
    if (previewStyles === false) {
      return '';
    }`
Which gets really confused by the typing. As does it in settings. So we'll have to put in `as string` in a couple of places at least.

Reply:
Okay, in that case, maybe don't change the setting and here just do this instead:
```
const previewStyles = Settings.getPreviewStyles(editor) || '';
```
That'll ensure the type is always a string here.

Reply:
Not fixed, as I noted in my other comment.

Reply:
This doesn't work. If we input a string at this point it will also include it for undefined, which is a different setting than false, and was causing unexpected behavior in the tests. The actual possible values are string, boolean, undefined, which all mean different things, apparently.